README.adoc

```diff
@@ -17,6 +17,9 @@
 :source-highlighter: prettify
 = Creating a MicroProfile application
 
+[.hidden]
+NOTE: This guide is deprecated. View the introductory guide to MicroProfile in https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices].
+
```

Comment:
NOTE: This guide is replaced by https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service
] and https://openliberty.io/guides/cdi-intro.html[Injecting dependencies into microservices] and has been archived. Use those guides instead.

Reply:
@sishida, please review the above.

Reply:
Updated.